src/current/v25.2/install-cockroachdb-linux.md

```diff
@@ -25,10 +25,21 @@ Use one of the options below to install CockroachDB. To upgrade an existing clus
 
 <div id="download-the-binary-linux" class="install-option">
   <h2 id="install-binary">Download the binary</h2>
-  {% include {{ page.version.version }}/misc/linux-binary-prereqs.md %}
+    <p>The CockroachDB binary for Linux requires <code>glibc</code>, <code>libncurses</code>, and <code>tzdata</code>, which are found by default on nearly all Linux distributions.</p>
   <ol>
     <li>
-      <p>Visit <a href="/docs/releases/index.html">Releases</a> to download the CockroachDB archive for the architecture of your Linux host. The archive contains the <code>cockroach</code> binary and the supporting libraries that are used to provide <a href="{% link {{ page.version.version }}/spatial-data-overview.md %}">spatial features</a>. Extract the archive and optionally copy the <code>cockroach</code> binary into your <code>PATH</code> so you can execute <a href="cockroach-commands.html">cockroach commands</a> from any shell. If you get a permission error, use <code>sudo</code>.</p>
+      <p>Visit <a href="/docs/releases/index.html">Releases</a> to download the CockroachDB archive for the architecture of your Linux host. The archive contains the <code>cockroach</code> binary and the supporting libraries that are used to provide <a href="{% link {{ page.version.version }}/spatial-data-overview.md %}">spatial features</a>. Follow the steps shown below to install the <code>cockroach</code> binary into your <code>PATH</code> so you can execute <a href="cockroach-commands.html">cockroach commands</a> from any shell.</p>
+      <ol>
+        <li>
+          Download the binary you want from the <a href="/docs/releases/index.html">Releases</a> page. In the following commands, replace <code>{VERSION}</code> with the version of CockroachDB you are installing, and replace <code>{ARCHITECTURE}</code> with <code>linux-amd64</code> for Intel, or with <code>linux-arm64</code> for ARM.
```

Comment:
I still think we should be more specific about the version syntax, because IMO it's extremely likely people will not prepend the `v`, at least (not sure if the liquid will still render with the code ticks around it):

```suggestion
          Download the binary you want from the <a href="/docs/releases/index.html">Releases</a> page. In the following commands, replace <code>{VERSION}</code> with the version of CockroachDB you are installing (e.g., `{{ page.version.version }}`, and replace <code>{ARCHITECTURE}</code> with <code>linux-amd64</code> for Intel, or with <code>linux-arm64</code> for ARM.
```